tests/ext/test_applicationbuilder.py

```diff
@@ -403,7 +416,9 @@ async def post_stop(app: Application) -> None:
         assert app.updater.update_queue is update_queue
         assert app.updater.bot is app.bot
         assert app.context_types is context_types
-        assert app.concurrent_updates == concurrent_updates
+        assert isinstance(app.update_processor, SimpleUpdateProcessor)
+        assert app.update_processor.max_concurrent_updates == expected.max_concurrent_updates
+        assert app.concurrent_updates == app.update_processor.max_concurrent_updates
```

Comment:
I see, using `is` here won't work because you don't pass the `expected` object itself to `concurrent_updates` but rather an integer. Then this is ofc fine. 👍 Please do however also add a test that ensure that passing a `SUP` object to `concurrent_updates` actually works with `is`